### ISSUE TITLE
Ace Combat: Terrible performance when looking into the sun on some hardware

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#12870]
A user reported that looking into the sun causes bad performance issues on Samsung Galaxy A50s.  It's expected that say a readback would slow things down but the slowdowns seems much worse than they should be, if it's that.

**Comment by unknownbrackets** [comment]
Assuming we don't have a GE dump?  That could at least confirm if it's a blending operation or something else.

-[Unknown]

**Comment by hrydgard** [comment]
Not yet, I have some videos from the user though, but they don't say much more than the text above really. It drops down to 6 fps from a pretty good framerate, it seems.

**Comment by LunaMoo** [comment]
Personally don't think the sun is any heavy, but can't really test difference as on my pc PPSSPP can't go above monitor max framerate despite it's VRR and I have Vsync turned off and drivers set to respect app setting. I recall that people with potato hardware had more problems with observing the ground than sky in this game.

 Either way I got 2 AC games around the time AC7 was released on steam so can provide dumps, one for each AC game showing the sun:
[AC_DUMP.zip](https://github.com/hrydgard/ppsspp/files/4577275/AC_DUMP.zip)

 Take note that a large drop in FPS when the sun shows up could be caused by texture replacement or texture scaling. It definitely caused some stutter for me.
